changelog.json

```
@@ -1,9 +1,13 @@
 {
   "releases": {
+    "2.7.0-beta6": [
+      "[Added] Add ability to cherry-pick merge commits - #11877",
+      "[Fixed] User can no longer get into an bad persistent cherry-picking drag state - #11875"
```

Sorry I just found another typo 😭 
```suggestion
      "[Fixed] User can no longer get into a bad persistent cherry-picking drag state - #11875"
```